benefit-finder/cypress/fixtures/benefits-eligibility.json

```diff
@@ -0,0 +1,114 @@
+{
+    "scenario_1_covid": {
+        "en": {
+            "param": {
```

Comment:
Please remove this and follow the pattern in scenario3

https://github.com/GSA/px-benefit-finder/pull/902/files#diff-ebf47abf1eaef31294ffdbbbc744fb0a6bf48dc3bc09af80080b41b5880d33f7R84-R101